JSON.md

```diff
@@ -0,0 +1,53 @@
+Bencodex JSON Representation
+===========================
+
+## Introduction
+
+Bencodex JSON Representation is a standard way to represent a given Bencodex value to valid JSON text to be interchanged.
+
+This specification defines a standard way to represent a Bencodex value to (a subset of) JSON and how to decode/encode them.
+
+### Conventions Used in This Document
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in BCP 14[^RFC-2119] when, and only when, they appear in all capitals, as shown here.
+
+## Prefixes
+
+Bencodex JSON Representation defines a set of prefixes that **SHALL** be used for JSON[^JSON] string value to denote a corresponding Bencodex type. The defined prefixes are following:
+
+- `0x` denotes a hexadecimal byte-string. The rest of the string **SHOULD** be parsed as a byte-string encoded in hexadecimal.
+  - The casing of the value **MAY** be ignored.
```

Comment:
IMHO the way to handle casing should be done on both side:

```suggestion
- `0x` denotes a hexadecimal byte string. The rest of the string **SHOULD** be parsed as a byte-string encoded in hexadecimal.
  - The hexadecimal byte string **SHOULD NOT** contain any upper letters.
  - The casing of the value **SHOULD** be ignored.
```

Reply:
I was thinking about some use-cases where the casings could be helpful such as [EIP-55](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-55.md) but as it is specified as `SHOULD` I agree with you.